### DISCUSSION TITLE
Seeking Help to Develop an Android App with MIDI Piano and Randomization Functionality"

### DISCUSSION THREAD

**Discussion by MonkuDu** [community/community#136309] [Programming Help]
### Select Topic Area

Question

### Body

Subject: Seeking Collaboration for Android App Development (Virtual Piano with Randomizer)

Hi everyone,

I'm working on creating an Android app that features a virtual piano with a piano roll similar to FL Studio. The app will include a recording feature and a randomizer button, allowing users to play and record notes, then randomize those notes in different patterns and time signatures, similar to BandLab's AI song starter.

I'm a music enthusiast with over 15 years of experience playing keyboard melodies, but I don’t have formal music theory knowledge. I can naturally play a series of notes in different patterns, and I'm looking for someone who can help me turn this idea into a fully functional app.

If you're interested in collaborating, please check out the repository on GitHub: [GitHub Repo Link]

I would love to work with someone who has experience in Android development, especially with MIDI connectivity and randomization algorithms.

Looking forward to hearing from you!

Thanks,

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @MonkuDu ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
